ports/stm32/boards/NUCLEO_WB55/rfcore_firmware.py

```diff
@@ -308,17 +380,20 @@ def _fus_fwdelete():
     return _run_sys_hci_cmd(_OGF_VENDOR, _OCF_FUS_FW_DELETE)
 
 
-def _fus_run_fwupgrade(addr):
+def _fus_run_fwupgrade():
     # Note: Address is ignored by the FUS (see comments above).
+    addr = _read_addr()
```

Comment:
Should this verify `addr` ? (Specifically that it's non-zero, mostly just as a sanity check)